src/fist_nonbond_force.F

```diff
@@ -234,8 +234,8 @@ SUBROUTINE force_nonbond(fist_nonbond_env, ewald_env, particle_set, cell, &
                   fac_ei = 0.5_dp*fac_ei
                   fac_vdw = 0.5_dp*fac_vdw
                END IF
-               ! decide which interactions to compute
-               IF (do_multipoles) THEN
+               ! decide which interactions to compute\b
```

Comment:
typo?

Reply:
Yes, sorry for that. Do you want me to correct it with another commit or you can overwrite it? Also, shall I add tests to it?

Reply:
Yes, just push another commit. I'll then squash them before merging. And tests are of course always welcome :-)

Reply:
OK Got it!!